src/diagnostics/diagnostic.jl

```diff
@@ -156,3 +156,29 @@ include("define_diagnostics.jl")
 
 # Default diagnostics and higher level interfaces
 include("default_diagnostics.jl")
+
+if pkgversion(ClimaDiagnostics) < v"0.2.13"
+    # Default diagnostic resolution given a Space (approximately one point per
+    # element)
+    function default_diagnostic_num_points(domain::SphericalShell)
```

Comment:
What do you think about just including this function in ClimaLand, and not using the one in ClimaDiagnostics?

In a way, it is ClimaLand's choice.

Reply:
My goal would be to remove both the functions completely eventually. This setting is now default in the NetCDFWriter, so we could remove both functions once we switch to use `0.2.13` as minimum compat.

Reply:
ok! sounds good. Is it worth making an issue to remind us to do so?

Reply:
Opened a more general one for ClimaCore as well.